plugins/SegmentEditor/tests/UI/SegmentSelectorEditor_spec.js

```diff
@@ -57,18 +57,6 @@ describe("SegmentSelectorEditorTest", function () {
         expect(await page.screenshotSelector(selectorsToCapture)).to.matchImage('3_segment_editor_edit_name');
     });
 
-    it("should show the segment editor's available segments dropdown", async function() {
-        var elem = await page.$('.available_segments a.dropList');
-        await elem.hover();
-        await page.click('.available_segments a.dropList');
-        expect(await page.screenshotSelector(selectorsToCapture)).to.matchImage('6_segment_editor_droplist');
-    });
-
-    it("should change segment when another available segment clicked in segment editor's available segments dropdown", async function() {
-        await (await page.jQuery('.ui-menu-item div:contains(Add new segment)')).click();
-        await page.waitForNetworkIdle();
-        expect(await page.screenshotSelector(selectorsToCapture)).to.matchImage('6_segment_editor_different');
-    });
```

Comment:
When removing UI tests, please also remove screenshots that are no longer used

Reply:
The screenshots are still there and should be removed:
* https://github.com/matomo-org/matomo/blob/matomo-17890-remove-segment-selector/plugins/SegmentEditor/tests/UI/expected-screenshots/SegmentSelectorEditorTest_6_segment_editor_droplist.png
* https://github.com/matomo-org/matomo/blob/matomo-17890-remove-segment-selector/plugins/SegmentEditor/tests/UI/expected-screenshots/SegmentSelectorEditorTest_6_segment_editor_different.png